src/parser/scanner_support.c

```diff
@@ -133,8 +198,17 @@ struct st_hint_msg
     check_buffer (snprintf (NULL, 0, fmt, str1, str2));
     m_used += sprintf (msg_ptr + m_used, fmt, str1, str2);
   }
+  void addendstring ()
```

Comment:
```suggestion
  void add_end_string ()
```

Reply:
I will edit according to your comments. Thank you